### ISSUE TITLE
$in behavior not working as expected

### ISSUE THREAD

**Issue by kerfab** [kapouille/mongoquery#14]
Hello everyone,

I encountered an issue while working with `$in`, it is not working as expected (the way MongoDB does work). 

The following should have returned `True` instead of `False`:
 
```
>>> from mongoquery import Query
>>> o = {'a': {'b': ['test1', 'test2']}}
>>> q = Query({'a.b': {'$in': ['test2']}})
>>> q.match(o)
False
```

Would it be okay to update mongoquery so that `$in`/`$nin` follow MongoDB's specifications?

**Comment by kapouille** [comment]
But of course @kerfab ! 
As you can tell we have not stressed the implementation as much as you have, so anything that makes it behave more in-line with the "official" implementation is better.

**Comment by kerfab** [comment]
Very good! :+1: I will see that "$in" is upgraded as soon as I have time for it, which is within a week or two

**Comment by kerfab** [comment]
This is fixed in version 1.3.1! Enjoy.
